stl/inc/future

```diff
@@ -409,7 +409,7 @@ protected:
     }
 
 public:
-    conditional_t<is_default_constructible_v<_Ty>, _Ty, _Result_holder<_Ty>> _Result;
+    conditional_t<is_default_constructible_v<_Ty>, _Ty, _Result_holder<_Ty>> _Result{};

```

Comment:
`_Ty` is a user-provided type, we should avoid brace-init. And this needs to compile in C++14, so we shouldn't `= `conditional_t<...>()` (even if we did want to repeat that enormous typename). We should just add a member-initializer to the single constructor.

Reply:
I'll make this change.